docs/experimental-features/beaconing-detection.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[beaconing-detection]]
+== Beaconing detection
+
+This feature provides an early warning system for command and control (C2) beaconing activity. It monitors network traffic for indicators of compromise (IOCs), and provides analytics to aid your threat hunting.
+
+[discrete]
+=== Deploy the Beaconing detection package
+
+To deploy Beaconing detection in your environment, follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/beaconing.md[these steps].
+
+The release package includes dashboards for monitoring beaconing activity in your environment. Users can review signals via a Lens dashboard called `Beaconing discovery`.
+
+[role="screenshot"]
+image::images/beaconing-detection-1.png[]
+
+[discrete]
+=== Feature details
+
+The feature uses a {ref}/transforms.html[transform] to segment network data by host and process name, then runs scripted metric aggregations on the host-process name pairs. For a given time window, the scripted metric aggregation checks each partition for the following:
+
+* Periodicity of signals, accounting for jitter.
+* Low variation of bytes sent from source to destination.
+* Low variation of bytes sent from destination to source.
+
+The transform also filters out trusted applications and IPs in an attempt to reduce false positives. The transform outputs information about the process and host indicators, and the detection. It runs every hour to poll for new beacons.
```

Comment:
Slight revision to avoid the awkward comma. Also, is this referring to the [trusted applications](https://www.elastic.co/guide/en/security/current/trusted-apps-ov.html) feature in Elastic Security? If it's a separate thing, maybe use a different term to avoid confusion with the feature.
```suggestion
The transform also filters out trusted applications and IPs in an attempt to reduce false positives. The transform outputs information about the detection and the process and host indicators. It runs every hour to poll for new beacons.
```